packages/transport/e2e/tests/bridge.test.ts

```diff
@@ -31,173 +29,159 @@ describe('bridge', () => {
         TrezorUserEnvLink.disconnect();
     });
 
-    // there might be more versions of bridge out there, see https://github.com/trezor/webwallet-data/tree/master/bridge
-    // but they are not available from trezor-user-env, see https://github.com/trezor/trezor-user-env/tree/master/src/binaries/trezord-go/bin
-    ['2.0.26', '2.0.27', undefined].forEach(bridgeVersion => {
```

Comment:
this is the only change, removing forEach. A bigger change, yes, but I don't think it makes sense to test legacy versions of legacy bridge anymore. And I am adding tests for node-bridge in  #13493 where only 1 legacy version of legacy bridge will still be tested